_posts/2021-03-29-Michelle Obama in The Daily Princetonian.md

```diff
@@ -0,0 +1,172 @@
+        </p>
+
+        <p>For the three women,
+          <quotation>"the best thing to do"</quotation> had developed into almost a full-time commitment over the past semester. They began designing last summer, auditioned models before midterm break and sewed the fifty-plus creations themselves. All three expressed satisfaction with the outcome of their efforts,
+          <quote>with
+            <quoted>
+              <lastname>Chu</lastname>
+            </quoted> adding that she was
+            <quotation>"very relieved"</quotation> that the pressure was off.
+          </quote>
+        </p>
+
+        <header>Color me beautiful</header>
+
+        <p>Fashions were displayed around seasonal and evening themes — spring, summer, fall / winter and glittery "winelight" attire. While the designers' styles are very different, their creations were all striking and attractive. The designers used a multitude of bright and bold colors to highlight their unique pieces. As diverse as their use of colors, the three incorporated a wide variety of exotic motifs and off-</p>
+
+        <break continue="1">(Continued on page three)</break>
+      </column>
+
+      <media>
+        <centered>
+          <photo>...</photo>
+        </centered>
+        <mediacredit>
+          <name>Danny Shapiro</name> — Princetonian
+        </mediacredit>
+
+        <caption>
+          <student>
+            <name>Michelle Robinson</name>
+            <class year="1985">'85</class>
+          </student> models one of the outfits displayed Saturday night in a fashion show in
+          <place type="campus hall">Wu Hall</place> to aid Ethiopian famine victims.
+        </caption>
+      </media>
+
+    </border></item></column>
+  </page>
+
+  <page num="3">
+    <column num="1"><item num="3">
+      <headline>Fashion</headline>
+
+      <break resume="1">(Continued from page one)</break>
+
+      <p cont="true">beat styles. They fashioned designs that highlighted African and Chinese motifs, as well as dashing turbans, sashes, slits, and off-the-shoulder and sleeveless dresses. The creations varied in length but the designers seemed to favor the long and streamlined look.</p>
+
+      <header>Stand-outs</header>
+
+      <p>Some of the designs which stood out as highlights of the show included
+        <nickname>Trish</nickname>'s dresses of African batiste and her terry-cloth outfits,
+        <firstname>Yvonne</firstname>'s dark sleeveless dress dramatized by contrasting sashes and
+        <firstname>Karen</firstname>'s romantic organza skirt and lace top combination.
+      </p>
+
+      <p>Members of the audience expressed enthusiasm for the students' fashions, and gave the three designers a standing ovation at the end.
+        <quote>The presentation
+          <quotation>"gathered up a bunch of dynamic personalities, dynamic models — an all-over dynamic show,"</quotation> said
+          <quoted>
+            <student>
```

Comment:
You could use the `<student>` tag in the same way I suggested that you use the `<person>` tag above. If you used `<student name="Student Name">` every time a student appears in the article (which it doesn't appear that you've done here, as far as I can tell), you could easily remove the name tags altogether. Probably your imagined researcher won't care to know, literally, how many times the name Michelle appears in The Prince. But they _will_ care about how many times the student Michelle Obama appears. TEI/XML allows you to add the metadata that actually makes the text "Michelle" or "Obama" useful to a researcher beyond a simple text search for every time the last name appears.